test_requirements.txt

```diff
@@ -0,0 +1,6 @@
+psycopg2-binary
```

Comment:
probably psycopg2 is not necesarry here. I'd say we use the dependencies from tox.ini